src/Orneholm.CognitiveWorkbench.Web/Views/Vision/ComputerVision.cshtml

```diff
@@ -54,6 +54,12 @@
                                         <input type="text" required class="cwb-input-remember form-control" placeholder="en" asp-for="ComputerVisionAnalyzeRequest.ImageOcrLanguage" name="ImageOcrLanguage" />
                                     </div>
                                 </div>
+                                <div class="col">
+                                    <div class="form-group">
+                                        <label asp-for="ComputerVisionAnalyzeRequest.ImageRecognizeTextMode">Image Recognize Text Mode</label>
+                                        <input type="text" required class="cwb-input-remember form-control" placeholder="Printed" asp-for="ComputerVisionAnalyzeRequest.ImageRecognizeTextMode" name="ImageRecognizeTextMode" />
```

Comment:
I'd usggest changing this into a dropdown, as it is a required field and it only can have two values.
The also remove the class `cwb-input-remember`.

Reply:
If you can pass it like that, the dropdown list is already implemented in another PR that will arrive next